json_diff_test.go

```diff
@@ -367,8 +365,8 @@ func TestShouldNotExplodeWhenAFieldsAreFromDifferentType(t *testing.T) {
 		"socios": "a",
 	}
 
-	expected := []string{"socios"}
-	actual := jsondiff.Diff(value, value2)
+	expected := []string{"socios", "socios.a"}
```

Comment:
Why socios and socios.a? Should be just socios.a 🤔

Reply:
because specially in this test case the field "socios" has different type. In value is an object and value2 is a string.

Reply:
Maybe if the type is different all the object is different and we need just return socios

Reply:
good idea! changed!